airbyte-integrations/connectors/destination-oracle-strict-encrypt/metadata.yaml

```diff
@@ -7,17 +7,23 @@ data:
   connectorSubtype: database
   connectorType: destination
   definitionId: 3986776d-2319-4de9-8af8-db14c0996e72
-  dockerImageTag: 0.2.0
+  dockerImageTag: 0.3.0
   dockerRepository: airbyte/destination-oracle-strict-encrypt
   githubIssueLabel: destination-oracle
   icon: oracle.svg
   license: ELv2
   name: Oracle
-  normalizationConfig:
-    normalizationIntegrationType: oracle
-    normalizationRepository: airbyte/normalization-oracle
-    normalizationTag: 0.4.1
   releaseStage: alpha
+  releases:
+    breakingChanges:
+      0.3.0:
```

Comment:
does conn ops rely on the major version bump to trigger breaking change notifs? or is just any version in this list?

Reply:
It does!, changing this back to `1.0.0`